PCbuild/regen.targets

```diff
@@ -50,23 +50,23 @@
           DependsOnTargets="FindPythonForBuild">
     <Message Text="Regenerate @(_PegenOutputs->'%(Filename)%(Extension)',' ')" Importance="high" />
     <!-- Specify python.gram with POSIX-like path because the argument gets written into the file verbatim -->
-    <Exec Command="set PYTHONPATH=Tools\peg_generator%0D%0A$(PythonForBuild) -m pegen -q c ./Grammar/python.gram Grammar\Tokens -o Parser\parser.c"
+    <Exec Command="set PYTHONPATH=Tools\peg_generator%0D%0A$(PythonForBuild) -m pegen -q c &quot;./Grammar/python.gram&quot; &quot;Grammar\Tokens&quot; -o &quot;Parser\parser.c&quot;"
```

Comment:
Let's leave it off the ones that don't need it. If there's no variable involved, there's no way for a space to show up.

Reply:
```suggestion
    <Exec Command="set PYTHONPATH=Tools\peg_generator%0D%0A$(PythonForBuild) -m pegen -q c ./Grammar/python.gram Grammar\Tokens -o Parser\parser.c"
```

Reply:
I think the item group in this file could shave off the ``$(PySourcePath)`` according to https://github.com/python/cpython/pull/121384/files#r1666595584 as well.